packages/server/src/utils/builders/heroku.ts

```diff
@@ -52,13 +53,14 @@ export const getHerokuCommand = (
 		application.project.env,
 	);
 
+	const builderVersion = env.HEROKU_STACK_VERSION || '24';
 	const args = [
 		"build",
 		appName,
 		"--path",
 		buildAppDirectory,
 		"--builder",
-		"heroku/builder:24",
+		`heroku/builder:${builderVersion}`,
```

Comment:
Hey, I think this should be better to handle via the UI, something like this, adding some field to the database and then that value use in the builder
![image](https://github.com/user-attachments/assets/e2a58d61-bcfc-4def-b4ee-9c84fed323c9)